maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -277,6 +279,26 @@ class Orchestra(
         return true
     }
 
+    private fun scrollUntilVisible(command: ScrollUntilVisibleCommand): Boolean {
+        val endTime = System.currentTimeMillis() + command.timeout
+        var error: MaestroException.ElementNotFound? = null
+        do {
+            try {
+                maestro.waitForAppToSettle()
+                findElement(command.selector, 500)
+                return true
+            } catch (ignored: MaestroException.ElementNotFound) {
```

Comment:
Should we also be catching timeout exception. There could be a case where timeout configured is not sufficient and we should let user know if timeout needs to be adjusted

Reply:
There's no timeout exception. If the element is not found, only `ElementNotFound` is thrown